### ISSUE TITLE
QM comes unpinned when last viewed panel is no longer present

### ISSUE THREAD

**Issue by ssnepenthe** [johnbillion/query-monitor#620]
Hopefully that title makes sense...

Normally QM stays open as you browse a site. However if you view a panel that isn't guaranteed to be present on one page and then browse to another page where that panel is not present, QM closes entirely.

Example: browse to a single post page, open the QM blocks panel, then browse to an archive page. QM should close.

What would you think about falling back to the overview panel in this case?

**Comment by johnbillion** [comment]
Yeah this is sort of a feature originally intended for when you're viewing the PHP errors panel. If there are no errors then is disappears, if there are some then it pops up again. Works for the Logs and Timings panels too.

It doesn't make much sense for most panels though. Showing the Overview is the obvious choice for a fallback but it might be weird if it switched back to the Blocks panel again if you navigate back to a single post. I'll have a think.
